test/inductor/test_torchinductor_opinfo.py

```diff
@@ -536,6 +784,89 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
     ("var", "cuda"): {f16},
     ("var_mean", "cuda"): {f16},
     ("xlogy", "cuda"): {f16},
+    ("__rdiv__", "xpu"): {f16},
```

Comment:
Could we add xpu as a different section rather than flat the ops?

Reply:
I'll split them into different sections